cxtool/cx-integrations.sh

```diff
@@ -0,0 +1,454 @@
+#!/bin/bash
+
+generate=false
+deploy=false
+namespace=default
+endpoint=api.eu2.coralogix.com
+appdynamic=true
+subdynamic=true
+platform=kubernetes
+outputdir=output
+destination=local
+
+function main {
+
+  function upload {
+    if [ "$destination" == "s3" ]; then
+      if [ -z "$bucket" ]; then echo "bucket name is empty!"; exit 1; fi
+      if [ -z "$bucketpath" ]; then 
+        aws s3 mv ./$outputdir/$integration-manifests.yaml s3://$bucket/$integration-manifests.yaml 2>&1
+        if [ $? -ne 0 ]; then 
```

Comment:
What is the error in such a case?

Reply:
The error that is coming from AWS, the real error, I think its much more helpful rather than throwing some message